File: php/Car.php
<?php

include_once "Account.php";

  class Car {
   public $id = int;
   public $license = string;
   public $driver;
   public $passanger = int;

   public function __construct($license, $driver){
     $this->license = $license;
     $this->driver = $driver;
   }

   public function printDataCar(){
     echo("License: " . $this->license);
     echo("Driver: " . $this->driver->name);
   }
  }

File: java/Car.java
class Car{
  Integer id;
  String license;
  Account driver;
  private Integer passenger;

  public Car(String license, Account driver) {
    this.license = license;
    this.driver = driver;
  }

  void printDataCar() {
    if(passenger!= null){
      System.out.println("License: " + license + "; Name driver: " + driver.name + "| Passenger" + passenger);
    }
  }

  public Integer getPassenger(){
    return passenger;
  }

  public void setPassenger(Integer passenger){
    if(passenger == 4){
      this.passenger = passenger;
    }else{
      System.out.println("Most be mayor to 4");
    }
  }

}